GitVersionCore.Tests/IntegrationTests/GitFlow/DevelopScenarios.cs

```diff
@@ -12,7 +12,8 @@ public void WhenDevelopBranchedFromMaster_MinorIsIncreased()
         {
             fixture.Repository.MakeATaggedCommit("1.0.0");
             fixture.Repository.CreateBranch("develop").Checkout();
-            fixture.AssertFullSemver("1.1.0-unstable.0+0");
+            fixture.Repository.MakeACommit();
+            fixture.AssertFullSemver("1.1.0-unstable.1+0");
```

Comment:
`develop` is inconsistent. Other tags start counting at 1, develop at 0. Making the code generic means develop now starts counting it's tag at 1